src/test/java/api/items/ItemMarkWithdrawnApiTest.java

```diff
@@ -166,82 +138,50 @@ public void shouldNotReopenExpiredRequests(String requestStatus) throws Exceptio
     "Closed - Unfilled"
   })
   public void shouldNotReopenClosedRequests(String requestStatus) throws Exception {
-    IndividualResource instance = instancesClient.create(smallAngryPlanet(UUID.randomUUID()));
-    UUID holdingId = holdingsStorageClient.create(
-      new HoldingRequestBuilder()
-        .forInstance(instance.getId()))
-      .getId();
-
-    IndividualResource createdItem = itemsClient.create(new ItemRequestBuilder()
-      .forHolding(holdingId)
+    final IndividualResource createdItem = itemsClient.create(new ItemRequestBuilder()
+      .forHolding(holdingsRecord.getId())
       .withStatus("Awaiting pickup")
       .canCirculate());
 
     final IndividualResource request = createRequest(createdItem.getId(),
       requestStatus, DateTime.now(DateTimeZone.UTC).plusHours(1));
 
-    final JsonObject withdrawnItem = markItemWithdrawn(createdItem.getId()).getJson();
-    assertThat(withdrawnItem.getJsonObject("status").getString("name"),
-      is("Withdrawn"));
+    assertThat(markItemWithdrawn(createdItem).getJson(), isWithdrawn());
+    assertThat(itemsClient.getById(createdItem.getId()).getJson(), isWithdrawn());
 
-    assertThat(itemsClient.getById(createdItem.getId()).getJson()
-      .getJsonObject("status").getString("name"), is("Withdrawn"));
-
-    assertThat(requestStorageClient.getById(request.getId()).getJson()
-      .getString("status"), is(requestStatus));
+    assertThat(requestStorageClient.getById(request.getId()).getJson(),
+      hasStatus(requestStatus));
   }
 
   @Test
   public void canMarkWithdrawnItemAsMissing() throws Exception {
-
-    IndividualResource instance = instancesClient.create(smallAngryPlanet(UUID.randomUUID()));
-    UUID holdingId = holdingsStorageClient.create(
-      new HoldingRequestBuilder()
-        .forInstance(instance.getId()))
-      .getId();
-
-    IndividualResource createdItem = itemsClient.create(new ItemRequestBuilder()
-      .forHolding(holdingId)
+    final IndividualResource createdItem = itemsClient.create(new ItemRequestBuilder()
+      .forHolding(holdingsRecord.getId())
       .withStatus("Available")
       .canCirculate());
 
-    final IndividualResource request = createRequest(createdItem.getId(),
+    createRequest(createdItem.getId(),
```

Comment:
Why is a request important for this scenario?

Reply:
I don't think that it is required at all. I'll remove this part.